var/spack/repos/builtin/packages/regiodesics/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Regiodesics(CMakePackage):
+
+    git = "ssh://bbpcode.epfl.ch/viz/Regiodesics"
+
+    version('0.1.0', tag='0.1.0', submodules=True)
+
+    depends_on('cmake@3.1:', type='build')
+    depends_on('ninja', type='build')
+    depends_on('bbptestdata')
+    depends_on('boost@1.65.0')
```

Comment:
can we relax this? I assume this builds with other boost versions as well?

Reply:
This should be revisited with the new deployment release.

Reply:
@ppodhajski do you absolutely have to have 1.65?